uber/model_checks.py

```diff
@@ -37,6 +37,30 @@ def has_email_address(account):
 
 
 @validation.Group
```

Comment:
This is the one change in functionality here - by moving these to the `@validation.Group` class, these are now run every time a group is saved, including in the admin form. Hopefully that won't be too inconvenient.